packages/fx-core/src/common/local/localEnvManager.ts

```diff
@@ -29,6 +30,7 @@ import { ProjectSettingsHelper } from "./projectSettingsHelper";
 import { LocalCertificate, LocalCertificateManager } from "./localCertificateManager";
 import { DepsManager } from "../deps-checker/depsManager";
 import { LocalStateProvider } from "../localStateProvider";
+import { EnvInfoV2 } from "@microsoft/teamsfx-api/build/v2";
```

Comment:
`v2.EnvInfoV2` or `EnvInfoV2`? May save one import here.

Reply:
"@microsoft/teamsfx-api/build/v2" may not be a good style since the `build` output folder can be changed.
Could `EnvInfoV2` be imported directly from "@microsoft/teamsfx-api"? Otherwise `v2.EnvInfoV2` is also acceptable.

Reply:
Thanks for the advice, changed to ```v2.EnvInfoV2```.